boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -145,6 +145,32 @@ arduino_i2c: &i2c1 {
 	pinctrl-names = "default", "sleep";
 };
 
+&spi4 {
+	compatible = "nordic,nrf-spim";
+	status = "okay";
+	pinctrl-0 = <&spi4_default>;
+	pinctrl-1 = <&spi4_sleep>;
+	pinctrl-names = "default", "sleep";
+	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+	mx25r64: mx25r6435f@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <8000000>;
```

Comment:
Why not 33 MHz?

Reply:
Wow sorry, I didn't see that you edited your comment and previously it was 8.